apps/system/test/marionette/software_home_file_open_error_test.js

```diff
@@ -48,34 +48,69 @@ marionette('Software Home Button - File Open Error', function() {
     system.gotoBrowser(url);
 
     // Save the file.
-    actions.longPress(client.helper.waitForElement('a'), 1).perform();
+    longPress(client.helper.waitForElement('a'));
     client.switchToFrame();
     system.appContextMenuSaveLink.click();
 
     // Tap on the toaster to open the download.
     // We could also open this from settings or the utility tray if needed.
-    var toasterTitle;
-    client.waitFor(function() {
-      toasterTitle = client.helper.waitForElement(
-        '#notification-toaster.displayed .toaster-title');
-      return toasterTitle.text().indexOf('Download complete') !== -1;
-    });
-    toasterTitle.tap();
+    getToasterTitle().tap();
```

Comment:
This name could be a little more semantic. Here we are getting the `Download Complete` notification toaster, and then tapping. So maybe we should call it `getDownloadCompleteToast` or something.